src/context/products/ProductContext.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { createContext, useContext, useState, useEffect } from "react";
+import axiosInstance from "../../utils/axios-instance";
+const ProductContext = createContext([]);
+const ProductProvider = ({ children }) => {
+  const [products, setProducts] = useState([]);
+  
+  useEffect(() => {
+    (async () => {
+      try {
+        const { data } = await axiosInstance.get("/products");
+        console.log(data);
+        setProducts(data.products);
+      } catch (error) {
+        console.log(error);
```

Comment:
remove console logs

Reply:
Thanks, I will push the changes in next commit